<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(231, 254, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Book Bed</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_7" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QWidget" name="widget_11" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>13</pointsize>
                 <weight>9</weight>
                 <italic>true</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 75 italic 13pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_12" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLineEdit" name="lineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
color:rgba(0,0,0,240);
padding-bottom:7px;</string>
               </property>
               <property name="placeholderText">
                <string>First name   Middle Name    Last Name</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QWidget" name="widget_13" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>13</pointsize>
                 <weight>9</weight>
                 <italic>true</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 75 italic 13pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Email</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_14" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLineEdit" name="lineEdit_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
color:rgba(0,0,0,240);
padding-bottom:7px;</string>
               </property>
               <property name="placeholderText">
                <string>eg. abc@gmail.com</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QWidget" name="widget_15" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>13</pointsize>
                 <weight>9</weight>
                 <italic>true</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 75 italic 13pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Date</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_16" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QLineEdit" name="lineEdit_3">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
color:rgba(0,0,0,240);
padding-bottom:7px;</string>
               </property>
               <property name="placeholderText">
                <string>DD/MM/YY</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QWidget" name="widget_17" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>13</pointsize>
                 <weight>9</weight>
                 <italic>true</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 75 italic 13pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Gender</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_18" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QLineEdit" name="lineEdit_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
color:rgba(0,0,0,240);
padding-bottom:7px;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QWidget" name="widget_19" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>13</pointsize>
                 <weight>9</weight>
                 <italic>true</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 75 italic 13pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Mobile Number</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_20" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <widget class="QLineEdit" name="lineEdit_6">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
color:rgba(0,0,0,240);
padding-bottom:7px;</string>
               </property>
               <property name="placeholderText">
                <string>10 digit number</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_8" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QWidget" name="widget_21" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Bed Type</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_22" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <item>
              <widget class="QLineEdit" name="lineEdit_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
color:rgba(0,0,0,240);
padding-bottom:7px;</string>
               </property>
               <property name="placeholderText">
                <string>Single Bed or ICU Bed</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">font: 75 italic 8pt &quot;Times New Roman&quot;;color:red;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_9" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item alignment="Qt::AlignHCenter">
           <widget class="QWidget" name="widget_23" native="true">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_16">
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>17</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>11</pointsize>
                 <weight>9</weight>
                 <italic>true</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Confirm&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton#pushButton{
	font: 75 italic 11pt &quot;Times New Roman&quot;;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 226));
	color: rgba(255,255,255,210);
	border-radius:5px;
}
QPushButton#pushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButton#pushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(150, 123, 111, 255);
}
</string>
               </property>
               <property name="text">
                <string>Confirm</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_24" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>30</height>
                </size>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Back&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton#pushButton_2{
	font: 75 italic 11pt &quot;Times New Roman&quot;;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 226));
	color: rgba(255,255,255,210);
	border-radius:5px;
}
QPushButton#pushButton_2:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButton#pushButton_2:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(150, 123, 111, 255);
}
</string>
               </property>
               <property name="text">
                <string>Back</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_10" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/image/Doctor.jpeg);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">background-image: url(:/image/Doctor.jpeg);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="images.qrc">:/image/Doctor.jpeg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../123/images.qrc"/>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
